.github/workflows/nomad.yaml

```diff
@@ -55,6 +55,9 @@ jobs:
 
           - scenario-name: write_validated_must_get_agent_activity
             required-nodes: 2
+
+          - scenario-name: zero_arc_create_data
+            required-nodes: 2
```

Comment:
2 nodes really isn't that meaningful, we could get this information from a sweettest. I think we should start moving away from the dev/smoke test values and start picking values for these tests that make use of the nodes we have available.

Reply:
Okay, I increased it to 10, given that we currently have 12 nomad clients available and to keep some margin if some of them might go offline every now and then.